java-client/src/test/java/org/opensearch/client/opensearch/integTest/OpenSearchJavaClientTestCase.java

```diff
@@ -154,7 +154,7 @@ protected String getTestRestCluster() {
     protected void wipeAllOSIndices() throws IOException {
         final IndicesResponse response = adminJavaClient()
             .cat()
-            .indices(r -> r.expandWildcards(ExpandWildcard.All));
+            .indices(r -> r.columnNames("index,creation.date").expandWildcards(ExpandWildcard.All));
```

Comment:
I think we changed `columnNames` to `headers`, right?

Reply:
right, fixed it